patches/server/0969-Added-the-ability-to-disable-non-player-map-markers.patch

```diff
@@ -0,0 +1,49 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: poonkje112 <aaron.knoop@live.nl>
+Date: Sun, 26 Mar 2023 04:17:53 +0200
+Subject: [PATCH] Added the ability to disable non-player map markers
+
+
+diff --git a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+index 8d442c5a498ecf288a0cc0c54889c6e2fda849ce..772cc473de64076dcb11b84f48fa08ab1d7ef167 100644
+--- a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
++++ b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+@@ -286,5 +286,6 @@ public class GlobalConfiguration extends ConfigurationPart {
+         public boolean lagCompensateBlockBreaking = true;
+         public boolean useDimensionTypeForCustomSpawners = false;
+         public boolean strictAdvancementDimensionCheck = false;
++        public boolean showMapMarkers = true;
+     }
+ }
+diff --git a/src/main/java/net/minecraft/world/level/saveddata/maps/MapItemSavedData.java b/src/main/java/net/minecraft/world/level/saveddata/maps/MapItemSavedData.java
+index 50348a5eec1e2ffabfc8405d708461376a162913..e7d5eaca95b7695460a8f48483fd80bb3fac9291 100644
+--- a/src/main/java/net/minecraft/world/level/saveddata/maps/MapItemSavedData.java
++++ b/src/main/java/net/minecraft/world/level/saveddata/maps/MapItemSavedData.java
+@@ -11,6 +11,7 @@ import java.util.List;
+ import java.util.Map;
+ import java.util.Objects;
+ import javax.annotation.Nullable;
++
+ import net.minecraft.core.BlockPos;
+ import net.minecraft.nbt.CompoundTag;
+ import net.minecraft.nbt.ListTag;
+@@ -44,6 +45,10 @@ import org.bukkit.craftbukkit.map.CraftMapView;
+ import org.bukkit.craftbukkit.util.CraftChatMessage;
+ // CraftBukkit end
+ 
++// Paper start
++import io.papermc.paper.configuration.GlobalConfiguration;
++// Paper end
++
```

Comment:
Remove all this diff. Use a fully qualified import for `GlobalConfiguration` and remove the newline added.